Contributing.md

```diff
@@ -205,5 +212,3 @@ Pull requests are typically reviewed within 10 business days.
 [GitHub Home]: http://github.com
 [GitHub Help]: http://help.github.com/
 [Set up Git]: https://help.github.com/articles/set-up-git/
-[Daring Fireball - Markdown]: http://daringfireball.net/projects/markdown/
-[Daring Fireball]: http://daringfireball.net/
```

Comment:
It seems like the Daring Fireball entry is referenced by the "More resources" section so you may want to keep this entry or remove the resource statement

Reply:
I'll remove the resource. Good call.